include/mbedtls/cipher.h

```diff
@@ -1,12 +1,14 @@
-/**
+
+ /**
  * \file cipher.h
  *
- * \brief Generic cipher wrapper.
- *
- * \author Adriaan de Jong <dejong@fox-it.com>
- */
+ * \brief The generic cipher wrapper.
+*/
+
 /*
- *  Copyright (C) 2006-2015, ARM Limited, All Rights Reserved
+ * \author Adriaan de Jong <dejong@fox-it.com>
```

Comment:
Blocker: there is something that looks like a Doxygen directive in a comment that is not processed by Doxygen. This is caught as an error by one of our validation scripts (`tests/scripts/check-doxy-blocks.pl`).

We've done a global search-and-replace to separate the copyright block, keeping the `\author` directive in the typeset documentation. @RosZad Going forward, please use the current state of the `development` branch as a basis for your edits.